modules/odc-importing-codebase-from-git-to-create-application.adoc

```diff
@@ -21,10 +21,11 @@ The following procedure walks you through the *From Git* option in the *Develope
 * *Context Dir* to specify the subdirectory for the application source code you want to use to build the application.
 * *Source Secret* to create a *Secret Name* with credentials for pulling your source code from a private repository.
 
-. Optional: You can import a `Devfile`, a `Dockerfile`, or a `Builder Image` through your Git repository to further customize your deployment.
-* If your Git repository contains a `Devfile`, a `Dockerfile`, or a `Builder Image`, it is automatically detected and populated on the respective path fields. If a `Devfile`, a `Dockerfile`, or a `Builder Image` are detected in the same repository, the `Devfile` is selected by default.
+. Optional: You can import a `Devfile`, a `Dockerfile`, `Builder Image` or a `Serverless Function` through your Git repository to further customize your deployment.
+* If your Git repository contains a `Devfile`, a `Dockerfile`, `Builder Image`, or a `func.yaml`, it is automatically detected and populated on the respective path fields. If a `Devfile`, a `Dockerfile`, or a `Builder Image` are detected in the same repository, the `Devfile` is selected by default. If `func.yaml` is detected in the Git repository, the *Import Strategy* changes to `Serverless Function`.
 * To edit the file import type and select a different strategy, click *Edit import strategy* option.
 * If multiple `Devfiles`, a `Dockerfiles`, or a `Builder Images` are detected, to import a specific instance, specify the respective paths relative to the context directory.
+* You can also create a serverless function by clicking *Create Serverless function* in the *+Add* view using the Git repository URL.
```

Comment:
"Also" can imply either "alternatively" or "in addition to". It's not clear to me which of these two meanings is intended here. And if this point is not related to any previous point, then I'd suggest dropping "also" because this point can exist on its only.

It's not clear to me as a reader whether

A. I can create a serverless function by clicking *Create Serverless function* in the *+Add* view.

B.I can create a serverless function by using the Git repository URL.

C. I can create a serverless function by clicking *Create Serverless function* in the *+Add* view [as some combination with] using the Git repository URL.

Reply:
@max-cx. I agree to what you're saying. I'm replacing the placement of this sentence (for continuation) instead of adding it as the last pointer. IMO it makes more sense there. Just to avoid confusion, I'm going to remove the word "also" and add "alternatively" to the sentence. Thanks for pointing this out.